Rewrite docs (#369)
DevExpress/DevExtreme.AspNet.Data: docs/server-side-configuration.md

## Custom Model Binder

> The server needs a custom model binder that receives data loading options from the client and binds them to the model. Refer to the links below to learn how to implement it.

Reviewer: ```suggestion
The server needs a custom model binder that receives data loading options from the client and binds them to the model. Refer to the links below for instructions on how to implement it.
```

- [ASP.NET Core](https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/net/Sample/DataSourceLoadOptions.cs)
- [Web API](https://github.com/DevExpress/devextreme-examples/blob/17_2/datagrid-webapi/datagrid-webapi/DataSourceLoadOptions.cs)
- [ASP.NET MVC 5](https://github.com/DevExpress/devextreme-examples/blob/17_2/datagrid-mvc5/datagrid-mvc5/DataSourceLoadOptions.cs)

> **NOTE:** If you use `DevExtreme.AspNet.Data` along with [DevExtreme-based ASP.NET Core controls](https://docs.devexpress.com/AspNetCore/400263) or [DevExtreme ASP.NET MVC 5 controls](https://docs.devexpress.com/DevExtremeAspNetMvc/400943/), you do not need to implement the custom model binder, because these libraries already include required binders.

Reviewer: ```suggestion
**NOTE:** If you use `DevExtreme.AspNet.Data` with [DevExtreme-based ASP.NET Core controls](https://docs.devexpress.com/AspNetCore/400263) or [DevExtreme ASP.NET MVC 5 controls](https://docs.devexpress.com/DevExtremeAspNetMvc/400943/), you do not need to implement the custom model binder because it is included in these libraries.
```
